shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -531,18 +590,18 @@ - (void)sendInitialSettings {
 }
 
 - (std::weak_ptr<flutter::AccessibilityBridgeMac>)accessibilityBridge {
-  return _bridge;
+  return _viewRecord.bridge;
```

Comment:
Can we remove this method entirely?

Reply:
Besides potentially moving it to `FVC`, I don't think we can remove this method entirely. The `accessibilityBridge` is used by `FlutterPlatformNodeDelegate` and `FlutterPlatformNodeDelegateMac`, for example [here](https://github.com/flutter/engine/blob/88a8d9a36705b0331ab2bdf7fdf60a0ba69f59a2/shell/platform/common/flutter_platform_node_delegate.cc#L41). So many methods are used that exposing `AccessibilityBridgeMac` is probably better than hiding it and adding every method to `FVC`.

This also answers [your other question](https://github.com/flutter/engine/pull/38855#discussion_r1070169895): `_bridge` needs to be a `shared_ptr`, not a `unique_ptr`, so that other class can get a `weak_ptr` out of it. (If we use `unique_ptr` other class can only get a raw pointer... not a modern C++ solution.)

Reply:
This method has been moved to `FVC`.